src/lib/replication/repl_dev/mapper.cpp

```diff
@@ -0,0 +1,116 @@
+#include <iomgr/iomgr_timer.hpp>
+#include <sisl/logging/logging.h>
+#include <sisl/fds/utils.hpp>
+#include <sisl/fds/vector_pool.hpp>
+
+#include "repl_dev/raft_state_machine.h"
+#include "repl_dev/raft_repl_dev.h"
+
+namespace homestore {
+repl_req_ptr_t RaftReplDev::follower_create_req(repl_key const& rkey, sisl::blob const& user_header,
+                                                sisl::blob const& user_key) {
```

Comment:
do we need to remove this mapper.cpp? it seems follower_create_req/notify_after_data_written functions are duplicated with the implementation in raft_repl_dev.cpp and map_fetch_write_pop() is never used.

Reply:
Yes I thought I removed it in next commit, if not, will remove that.